packages/resolve-es/src/index.js

```diff
@@ -0,0 +1,17 @@
+export default (driver) => {
```

Comment:
There should be `({ driver })` (https://trello.com/c/R0yu1Vg8/95-resolve-eventstore-initial).